api/v1/service/resource/resource.go

```diff
@@ -205,9 +205,9 @@ func (s *service) ByCatalogKindName(ctx context.Context, p *resource.ByCatalogKi
 		}
 	}
 
-	// If minPipelinesVersion is passed then check for version compatible with pipelines version
-	if p.Minpipelinesversion != nil {
-		r = filterCompatibleVersions(r, *p.Minpipelinesversion)
+	// If pipelinesVersion is passed then check for version compatible with pipelines version
+	if p.Pipelinesversion != nil {
```

Comment:
shouldn't this be `p.PipelinesVersion` ?

Reply:
goa makes it title case `P`.
for `v` to be `V`, query param has to be changed from `pipelinesversion` to `pipelinesVersion` :)